README.md

```diff
@@ -37,8 +37,7 @@ see Build instructions below ❗❗
 
 ### Config file
 
-Configuration of the eVolver system including provisioned hardware and experiments can be expressed in a single yaml
-file, for example:
+Configuration of the eVolver system including provisioned hardware and experiments must be expressed in a single yaml, `evolver.yml` stored in the project root directory file, for example:
```

Comment:
This isn't strictly true as the filename is custom by changing [evolver.settings.AppSettings.CONFIG_FILE](https://github.com/ssec-jhu/evolver-ng/blob/main/evolver/settings.py#L25).

The env var ``EVOLVER_CONFIG_FILE`` can also be used.

I think we should note this in the docs.

Reply:
It obviously doesn't even have to be in the project root dir.

Reply:
updated the readme to make this clear.